tachikoma-it/src/test/kotlin/com/sourceforgery/tachikoma/auth/UserServiceSpek.kt

```diff
@@ -1,8 +1,5 @@
 package com.sourceforgery.tachikoma.auth
```

Comment:
No more Spek is nice! But shouldn't the class reflect Junit and be named UserServiceTest instead?